src/views/record-browser/query-builder/query-builder.scss

```diff
@@ -10,27 +10,20 @@
 		h4 {
 			color: $secondary;
 		}
-
-		button {
-			width: 100%;
-			margin-left: 0;
-			margin-bottom: $padding;
-		}
 	}
 
-	.add-filter-button {
+	.add-filter-button.md-button {
 		background: $blue-gray;
-		height: 22px;
-		width: 22px;
+		min-width: 22px;
+		min-height: 22px;
 		line-height: 22px;
-		text-align: center;
-		border: 0;
+		padding: 0;
 		border-radius: 50%;
-		color: white;
 		float: right;
 
-		span {
-			vertical-align: 2px;
+		i {
```

Comment:
- Use class selectors. For example, `.fa`, `.fa-plus`, or `.add-filter-button-icon`.
- The plus sign is a bit large. Use a `font-size` of 11 or 12 px.